### PR TITLE
🐛Dask: Handle when gateway is offline

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?

An external gateway may go offline. the director-v2 was outputting misleading error code in this case.

Bonus:
- remove log pollution of aio_pika, aiormq
- remove log pollution when creating computations

## Related issue/s
- related to https://github.com/ITISFoundation/osparc-issues/issues/885
- related to https://github.com/ITISFoundation/osparc-issues/issues/675
- related to https://github.com/ITISFoundation/osparc-simcore/issues/2945

### CODE REVIEW

File: services/director-v2/src/simcore_service_director_v2/api/routes/clusters.py [hunk 1]
```diff
@@ -175,15 +180,18 @@ async def get_cluster_details(
...
-    return cluster_details
+    try:
+        cluster_details = await _get_cluster_details_with_id(
+            settings=settings,
+            user_id=user_id,
+            cluster_id=cluster_id,
+            clusters_repo=clusters_repo,
+            dask_clients_pool=dask_clients_pool,
+        )
+        logger.debug("found followind %s", f"{cluster_details=!r}")
```
Comment (matusdrobuliak66) [inline services/director-v2/src/simcore_service_director_v2/api/routes/clusters.py:191]: "found followin**g**"

**Review by matusdrobuliak66 — APPROVED** [review]
:+1:

File: services/director-v2/src/simcore_service_director_v2/core/application.py [hunk 2]
```diff
@@ -95,6 +102,14 @@ def create_base_app(settings: Optional[AppSettings] = None) -> FastAPI:
...
     logger.debug(settings.json(indent=2))
 
+    # keep mostly quiet noisy loggers
+    quiet_level: int = max(
+        min(logging.root.level + LOG_LEVEL_STEP, logging.CRITICAL), logging.WARNING
+    )
+
+    for name in NOISY_LOGGERS:
+        logging.getLogger(name).setLevel(quiet_level)
+
```
Comment (GitHK) [inline services/director-v2/src/simcore_service_director_v2/core/application.py:112]: May I ask for this logic to me moved to the `packages/service-library/src/servicelib/rabbitmq_utils.py`
I'd like to reuse it in the dynamic-sidecar as well.

**Review by GitHK — APPROVED** [review]
Nice. Just one small request.

File: services/director-v2/src/simcore_service_director_v2/core/application.py (hunk 2 above)
Comment (sanderegg) [reply services/director-v2/src/simcore_service_director_v2/core/application.py:112]: https://github.com/ITISFoundation/osparc-simcore/blob/ccec5a6dae410413562f3ecc111598e130dc95ca/services/web/server/src/simcore_service_webserver/log.py
 for reference

**Review by pcrespov — APPROVED** [review]

**Pull request merged**